app/src/main/java/com/orange/ouds/app/ui/tokens/TokenCategoryDetailScreen.kt

```diff
@@ -46,35 +47,53 @@ fun TokenCategoryDetailScreen(tokenCategory: TokenCategory) {
                 Spacer(modifier = Modifier.height(OudsSpacingFixedKeyToken.Medium.value))
             }
 
-            items(tokens) { token ->
-                Row(
-                    modifier = Modifier
-                        .fillMaxWidth()
-                        .padding(horizontal = OudsSpacingFixedKeyToken.Medium.value, vertical = OudsSpacingFixedKeyToken.Shorter.value)
-                ) {
-                    when (tokenCategory) {
-                        is TokenCategory.Opacity -> tokenCategory.Illustration(opacity = token.value as Float)
-                        is TokenCategory.Elevation -> tokenCategory.Illustration(elevation = token.value as Dp)
-                    }
-                    Column(
+            items(tokenCategory.properties) { tokenProperty ->
```

Comment:
Using `TokenProperty` is OK for now, but when we will do grid tokens, we will also need these kind of sections for "Android: Extra Compact", "Android: Compact" and "Android: Medium", where "Android" is a property and "Extra Compact" is a concept in Figma. 

Maybe will we have to do something more generic because I'm afraid sections will not always be equivalent to a Figma property.

Reply:
I discussed with Benoît and we will not have to display all configurations (Extra compact, Compact, Medium). We will only display the tokens of the current configuration cause we have to reflect the use of tokens from a developer point of view.